force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -164,17 +223,22 @@ public with sharing class FlowControlLoopsCollections {
         // Initialize the result String
 
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
+        String finalString='';
+        for (Integer i=0 ; i<repeatCount; i++) {
+            if(i==repeatCount-1) {
+                finalString = finalString + inputString;
+            } else {
+                finalString = finalString + inputString + '; ';
+            }
```

Comment:
You can also wait until your loop is finished then remove the semi colon at the end